hw/lint/data/veriblelint.hjson

```diff
@@ -0,0 +1,21 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+    tool_srcs: []
+
+    // TODO(#1727): pass Verible config file to FuseSoC, once supported by Verible
+    // We rely on fusesoc to run lint for us
+    build_cmd:  "fusesoc "
+    build_opts: ["--cores-root {proj_root}",
+                 "run --target={flow} --tool={tool} --build-root={build_dir}",
+                 " {fusesoc_core} "]
+
+    // TODO(#1727): switch over to native structured tool output, once supported by Verible
```

Comment:
Let's please not link a catch-all issue here, that's impossible to track and action. Ideally, we can link a Verible issue?

Reply:
I've added it to the DVSIM tracker since the output parsing script is DVSIM specific, but I can also add a point here in this Verible-related issue: https://github.com/lowRISC/opentitan/issues/1342

Reply:
I've carried these TODOs over to the Verible Issue #1342